sh.loft.devpod.yaml

```diff
@@ -0,0 +1,52 @@
+id: sh.loft.devpod
+
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+
+command: "sh.loft.devpod"
+finish-args:
+  - --socket=wayland # Permission needed to show the window
+  - --socket=fallback-x11 # Permission needed to show the window
+  - --device=dri # OpenGL, not necessary for all projects
+  - --share=ipc
+  - --share=network
+  - --socket=ssh-auth
+  - --socket=gpg-agent
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
needs an exception. https://docs.flathub.org/docs/for-app-authors/linter#exceptions

Reply:
https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access is the error string.